models/Item.yaml

```diff
@@ -0,0 +1,132 @@
+type: object
+name: Item
+label: Item
+labelField: title
+fieldGroups:
+  - name: styles
+    label: Styles
+  - name: settings
+    label: Settings
+fields:
+  - type: enum
```

Comment:
Missing an `elementId` field. This allows every component to have a unique element id used as a fallback for css styling.

```
export default function HeroSection(props) {
    const cssId = props.elementId || null;
    
    return (
        <div
            id={cssId}
            className={classNames(
```